functests/1_performance/cpu_management.go

```diff
@@ -91,16 +93,50 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			Expect(conf).To(MatchRegexp(fmt.Sprintf(`"reservedSystemCPUs": ?"%s"`, reservedCPU)))
 
 			By("checking CPU affinity mask for kernel scheduler")
-			cmd = []string{"/bin/bash", "-c", "taskset -pc $(pgrep 'rcu_sched|rcu_preempt')"}
+			cmd = []string{"/bin/bash", "-c", "taskset -pc 1"}
```

Comment:
I miss the "why" in the commit message, I am sure the change is needed, but a reason will help more :)

Reply:
to have "more generic approach", will add it to commit message

Reply:
thanks